salt/runners/salt.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+'''
+Runner to call execution modules
+
+.. versionadded:: Carbon
+'''
+# import python libs
+from __future__ import absolute_import
+from __future__ import print_function
+import logging
+
+# import salt libs
+from salt.loader import minion_mods
+
+log = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def cmd(fun, args=None, kwargs=None):
```

Comment:
Thanks for the wonderful wonderful suggestion @rallytime. I was under the wrong assumption about `args` and `kwargs` parsing; but your test example above makes it clear. Sorry that the implementation surprised you / wasted time. I'll fix it.